src/ApiService/ApiService/onefuzzlib/Config.cs

```diff
@@ -262,6 +262,24 @@ await _containers.GetContainerSasUrl(x.Item2.Name, StorageType.Corpus, ConvertPe
             }
         }
 
+        if (definition.Features.Contains(TaskFeature.FunctionAllowlist)) {
+            if (task.Config.Task.FunctionAllowlist != null) {
+                config.FunctionAllowlist = task.Config.Task.FunctionAllowlist;
+            }
```

Comment:
tip: that this can be written

```suggestion
            if (task.Config.Task.FunctionAllowlist is string functionAllowList) {
                config.FunctionAllowlist = functionAllowlist;
            }
```

but don't bother updating all of these unless you really want 🙂

Reply:
Will do a follow-up pass!